clusterman/autoscaler/autoscaler.py

```diff
@@ -181,7 +181,11 @@ def run(self, dry_run: bool = False, timestamp: Optional[arrow.Arrow] = None) ->
             self.target_capacity_gauge.set(new_target_capacity, {"dry_run": dry_run})
             self.max_capacity_gauge.set(
                 self.pool_manager.max_capacity,
-                {"dry_run": dry_run, "alert_on_max_capacity": self.pool_manager.alert_on_max_capacity},
+                {
+                    "dry_run": dry_run,
+                    "alert_on_max_capacity": self.pool_manager.alert_on_max_capacity,
+                    "team": self.pool_manager.pool_owner,
```

Comment:
@gmdfalk actually, I was just talking to @EmanElsaban about another alert we wanted to use internally and we realized that having a `team` label on all of the clusterman metrics would be useful - thoughts on adding the label to everything else in this PR?

Reply:
@nemacysts No, probably makes sense.

In that case, i think we should be able to add the team label to `autoscaler._emit_requested_resource_metrics` and `drainer._emit_draining_metrics` to tag onto most if not all metrics emitted?